test/controllers/login_controller_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+Dir[Rails.root.join("test/**/*.rb")].each {|f| require f}
+
+# https://gist.github.com/kinopyo/1338738
+class LoginControllerTest < ActionController::TestCase
+  include OmniauthMacros
+
+  describe "POST #urs" do
+    before do
+      mock_auth_hash
+      request.env["devise.mapping"] = Devise.mappings[:user] # If using Devise
```

Comment:
not a huge deal, but this comment seems to be either unclear or unnecessary to me.

Reply:
Agree, removed it.